infrastructures/docker/infrastructure/src/test/java/org/eclipse/che/workspace/infrastructure/docker/DockerMachineSourceTest.java

```diff
@@ -84,32 +79,19 @@ public void testConstructors(
         new DockerMachineSource(repository).withTag(tag).withRegistry(registry).withDigest(digest);
     assertEquals(source1.getLocation(), location);
 
-    DockerMachineSource source2 = new DockerMachineSource(source1);
-    assertEquals(source2.getLocation(), location);
-    assertEquals(source2.getRegistry(), registry);
-    assertEquals(source2.getRepository(), repository);
-    assertEquals(source2.getTag(), tag);
-    assertEquals(source2.getDigest(), digest);
-
     DockerMachineSource source3 = new DockerMachineSource(repository);
     source3.setTag(tag);
     source3.setRegistry(registry);
     source3.setDigest(digest);
     assertEquals(source3.getLocation(), location);
   }
 
-  /** Check valid source type */
-  @Test(expectedExceptions = InternalInfrastructureException.class)
-  public void testInvalidSourceType() throws InternalInfrastructureException {
-    when(machineSource.getType()).thenReturn("invalid");
-    new DockerMachineSource(machineSource);
-  }
-
-  /** Check invalid format */
-  @Test(expectedExceptions = InternalInfrastructureException.class)
-  public void testInvalidFormat() throws InternalInfrastructureException {
-    when(machineSource.getType()).thenReturn("image");
-    when(machineSource.getLocation()).thenReturn("@image");
-    new DockerMachineSource(machineSource);
-  }
+  //  TODO: that should be checked in DockerMachineStarter
```

Comment:
Are you going to uncomment or move this test?

Reply:
I suppose there is no big reason to wrte tests for DockerMachineStarter right now, as it still have to be strongly refactored (as seen in the class TODO). So this test probably have to be removed now and rewrited lated.

Reply:
+1 to remove